docs/relationship-particles.md

```diff
@@ -0,0 +1,386 @@
+---
+nav_order: 6
+---
+
+# Relationship with systemd "particles"
+
+There is an excellent [vision blog entry](https://0pointer.net/blog/fitting-everything-together.html)
+that puts together a coherent picture for how a systemd (and [uapi-group.org](https://uapi-group.org/))
+oriented Linux based operating system can be put together, and the rationale for doing so.
+
+The "bootc vision" aligns with parts of this, but differs in emphasis and also some important technical details - and some of the emphasis and details have high level ramifications.  Simply stated: related but different.
+
+## System emphasis
+
+The "particle" proposal mentions that the desktop case is most
+interesting; the bootc belief is that servers are equally
+important and interesting.  In practice, this is not a real point
+of differentation, because the systemd project has done an excellent
+job in catering to all use cases (desktop, embedded, server) etc.
+
+An important aspect related to this is that the bootc project exists and must
+interact with many ecosystems, from "systemd-oriented Linux" to Android and
+Kubernetes. Hence, we would not explicitly compare with just ChromeOS, but also
+with e.g. [Kairos](https://kairos.io/) and many others.
+
+## Design goals
+
+Many of the toplevel design goals do overall align.  It is clear that e.g. Discoverable Disk Images and container image (OCI) align on managing systems in an image-oriented fashion.
```

Comment:
add (DDI) and replace `container image` with `Open Container Initiative` - and link both `Discoverable Disk Images` and Open Container Initiative here.

Reply:
Good suggestion, done